RestCommClient/Classes/Push/Binding.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * TeleStax, Open Source Cloud Communications
+ * Copyright 2011-2015, Telestax Inc and individual contributors
+ * by the @authors tag.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation; either version 3 of
+ * the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ * For questions related to commercial use licensing, please contact sales@telestax.com.
+ *
+ */
+
+
+#import <Foundation/Foundation.h>
+
+@interface Binding : NSObject
```

Comment:
How come we have Binding.h/m but no similar modules for account, credentials, etc?

Reply:
we don't need to have all the entities saved on SDK side.
All we need are Sid's. 
Binding object is something we create/update and we need all data just for it.

Reply:
Don't you also create account and credential entities?

Reply:
yes, but we dont use anything from it, except the sid's

Reply:
Ok let me check the whole thing in the second-pass review